pyresample/geometry.py

```diff
@@ -2772,40 +2771,49 @@ def get_geostationary_angle_extent(geos_area):
     aeq = 1 - req ** 2 / (h ** 2)
     ap_ = 1 - rp ** 2 / (h ** 2)
 
-    # generate points around the north hemisphere in satellite projection
-    # make it a bit smaller so that we stay inside the valid area
-    xmax = np.arccos(np.sqrt(aeq))
-    ymax = np.arccos(np.sqrt(ap_))
-    return xmax, ymax
+    x_angle = np.arccos(np.sqrt(aeq))
+    y_angle = np.arccos(np.sqrt(ap_))
+    return x_angle, y_angle
 
 
 def get_geostationary_bounding_box_in_proj_coords(geos_area, nb_points=50):
     """Get the bbox in geos projection coordinates of the valid pixels inside `geos_area`.
 
-    Notes:
-    - The first and last element of the output vectors are equal.
-    - If nb_points is even, it will return x and y vectors of length nb_points + 1.
```

Comment:
Is it a problem for any of the other interfaces (the bounds and polygon stuff) that the end points are not the same?

Reply:
The tests didn't break, so I think we're good. Also I always thought that was the way we did it in pyresample.